Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -367,7 +367,9 @@ def bottle_formula(f)
       mismatches = [:root_url, :prefix, :cellar, :rebuild].reject do |key|
         old_spec.send(key) == bottle.send(key)
       end
-      if old_spec.cellar == :any && bottle.cellar == :any_skip_relocation
+      if (old_spec.cellar == :any && bottle.cellar == :any_skip_relocation) ||
+         ([HOMEBREW_DEFAULT_CELLAR, LINUXBREW_DEFAULT_CELLAR].include?(old_spec.cellar) &&
```

Comment:
Pretty sure these variables don't exist yet and will need defined. I just used them as short-hand. See https://github.com/Homebrew/brew/blob/1af5f668146013f5b0d6ce1adac3a0f3adb21f0f/Library/Homebrew/dev-cmd/bottle.rb#L19-L24 and your last change in https://github.com/Homebrew/brew/commit/8ff7601a92b9200c660706d60e04aa63200baa73

Reply:
Right, good thing there is a code review. I found out there is actually `cellar = HOMEBREW_CELLAR` written earlier in the method, so I used this.